app/views/patients/show.html.erb

```diff
@@ -55,7 +55,9 @@
                         current_user: current_user,
                         translations: Assessment.new.translations,
                         authenticity_token: form_authenticity_token,
-                        jurisdiction_paths: @possible_jurisdiction_paths
+                        jurisdiction_paths: @possible_jurisdiction_paths,
+                        num_pos_labs: @num_pos_labs,
+                        calculatedSymptomOnset: @calculated_symptom_onset
```

Comment:
same here: can we make this snake case to be consistent with all other props coming from .erb files

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-38caf8aa9518c92857c2602080a58e672992506202b8d2b00adc4470dc9b10a2R60